### PR TITLE
Small adjustments to reflect RC0 deployment

### PR DESCRIPTION
**Pull request by asaj**
### Description
Changes to .env file, genesis block, automatically whitelist contract deployer if cGLD price set in migrations config, don't send as many txs in parallel for governance constitution.
### Tested

Tested as part of RC0 deployment
### Other changes

None

### Related issues

None
### Backwards compatibility

None

### CODE REVIEW

File: packages/protocol/migrations/24_governance.ts [hunk 1]
```diff
@@ -95,10 +92,8 @@ module.exports = deploymentForCoreContract<GovernanceInstance>(
       'Validators',
     ]
 
-    await Promise.all(
```
Comment (aslawson) [inline packages/protocol/migrations/24_governance.ts:98]: does removing this make the tx happen sequentially instead of in parallel? is this related to the ledger flakiness?

File: packages/protocol/migrations/09_stabletoken.ts [hunk 2]
```diff
@@ -60,10 +60,12 @@ module.exports = deploymentForCoreContract<StableTokenInstance>(
...
       const isOracle = config.stableToken.oracles.some((o) => eqAddress(o, fromAddress))
-      console.assert(
-        isOracle,
-        `Gold price specified in migration but ${fromAddress} not authorized as oracle`
-      )
+      if (!isOracle) {
+        console.warning(
+          `Gold price specified in migration but ${fromAddress} not explicitly authorized as oracle, authorizing...`
+        )
+        await sortedOracles.addOracle(stableToken.address, ensureLeading0x(fromAddress))
```
Comment (aslawson) [inline packages/protocol/migrations/09_stabletoken.ts:67]: are you doing this dynamically just so the contract deployer address doesn't have to be duplicated in both the the truffle config and migration config?

**Review by aslawson — APPROVED** [review]
LGTM -- though seems to not be building or installing dependencies...

File: packages/protocol/migrations/09_stabletoken.ts (hunk 2 above)
Comment (asaj) [reply packages/protocol/migrations/09_stabletoken.ts:67]: Yeah I saw the migrations fail b/c the "from" address wasn't explicitly authorized, which is a pain. It's easier to whitelist the oracle now and remove it later via governance.

File: packages/protocol/migrations/24_governance.ts (hunk 1 above)
Comment (asaj) [reply packages/protocol/migrations/24_governance.ts:98]: Correct, now it will be in series, which while slower seems to be more reliable
